ecl/hql/CMakeLists.txt

```diff
@@ -167,6 +167,9 @@ if (CMAKE_COMPILER_IS_GNUCC OR CMAKE_COMPILER_IS_CLANG)
 
   #Compiler complains about an unused function yyunput, and make sure that any warnings are not reported as errors
   set_source_files_properties (hqllex.cpp PROPERTIES COMPILE_FLAGS " -Wno-unused-function -Wno-error")
+
+  #Compiler complains about a variable is set but not used 
+  set_source_files_properties (hqlgram.cpp PROPERTIES COMPILE_FLAGS " -Wunused-but-set-variable -Wno-error")
```

Comment:
I think this should be
```
set_source_files_properties (hqlgram.cpp PROPERTIES COMPILE_FLAGS " -Wno-unused-but-set-variable")
```

Reply:
Ended up changing to:  `-Wno-error=unused-but-set-variable -Wno-error=class-memaccess`